tests/unit/workflows/test_workflows_tasks_refextract.py

```diff
@@ -93,8 +94,11 @@ def test_extract_journal_info_handles_year_an_empty_string():
     ]
 
 
-def test_extract_references_handles_unicode():
+def test_extract_references_outputs_valid_data():
     filename = pkg_resources.resource_filename(
         __name__, os.path.join('fixtures', '1704.00452.pdf'))
 
-    extract_references(filename)
+    extracted_references = extract_references(filename)
+    ref_schema = get_value(load_schema('hep'), 'properties.references')
+
+    validate(extracted_references, ref_schema)
```

Comment:
You're testing more than before, but you could be testing even more: as @michamos suggested (perhaps only IRL?), the fact that the first reference is correctly extracted.